### ISSUE TITLE
Modulelist instead of Sequential

### ISSUE THREAD

**Issue by ruotianluo** [ignacio-rocco/detectorch#4]
https://github.com/ignacio-rocco/detectorch/blob/master/lib/model/detector.py#L17

If you want to group layers, what about using ModuleList

**Comment by ignacio-rocco** [comment]
Hey, thanks. Sounds like a good idea.

I was not aware of nn.ModuleList.

Should we also do it for https://github.com/ignacio-rocco/detectorch/blob/master/lib/model/detector.py#L202?

In this case the modules don't have any parameters so maybe it's less necessary.

**Comment by ruotianluo** [comment]
I would suggest having that. When you are switching training mode, it will also affect all submodules. Although currently the proposal layer seems not care about the training mode at all, just it's good for later.

**Comment by ignacio-rocco** [comment]
Fixed in https://github.com/ignacio-rocco/detectorch/commit/eadb1189effe79d912ff98ce2cd65d4a6dfc0bdd
